sql/updates/world/master/2025_11_27_00_world.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM `spell_proc` WHERE `SpellId` IN (7384);
+INSERT INTO `spell_proc` (`SpellId`,`SchoolMask`,`SpellFamilyName`,`SpellFamilyMask0`,`SpellFamilyMask1`,`SpellFamilyMask2`,`SpellFamilyMask3`,`ProcFlags`,`ProcFlags2`,`SpellTypeMask`,`SpellPhaseMask`,`HitMask`,`AttributesMask`,`DisableEffectsMask`,`ProcsPerMinute`,`Chance`,`Cooldown`,`Charges`) VALUES
+(7384,0x00,4,0x00000000,0x00000000,0x00000000,0x00000000,0x0,0x0,0x0,0x2,0x0,0x8,0x0,0,0,0,1); -- Overpower
```

Comment:
requires limiting to Mortal Strike / Cleave via. SpellFamilyMask

* also missing new line at the end of file